src/net_permissions.cpp

```diff
@@ -104,11 +117,11 @@ bool NetWhitebindPermissions::TryParse(const std::string& str, NetWhitebindPermi
     return true;
 }
 
-bool NetWhitelistPermissions::TryParse(const std::string& str, NetWhitelistPermissions& output, bilingual_str& error)
+bool NetWhitelistPermissions::TryParse(const std::string& str, NetWhitelistPermissions& output, ConnectionDirection& output_connection_direction, bilingual_str& error)
```

Comment:
Not a big fan of adding extra out-parameters, I think `util::Result` would be more appropriate here (but also a bit of an overhaul so wouldn't be opposed to doing it in follow-up)